canopeum_frontend/src/components/social/AnnouncementCard.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react'
+import { Link } from 'react-router-dom'
+
+import SiteAnnouncementModal from '@components/social/site-modal/SiteAnnouncementModal'
+import type { PageViewMode } from '@models/types/PageViewMode.Type'
+import type { Announcement } from '@services/api'
+
+type Props = {
+  readonly announcement: Announcement,
+  readonly viewMode: PageViewMode,
+  readonly onEdit: (announcement: Announcement) => void,
+}
+
+const AnnouncementCard = ({ announcement, viewMode, onEdit }: Props) => {
+  const [isModalOpen, setIsModalOpen] = useState(false)
+
+  return (
+    <>
```

Comment:
Small detail to reduce indentation
```suggestion
  return <>
```